modules/developer_manual/examples/core/scripts/ruby/decline-pending-share.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+require 'uri'
+
+base_uri = '{oc-examples-server-url}/ocs/v1.php/apps/files_sharing/api/v1/'
+uri = URI("#{base_uri}/shares/115468")
```

Comment:
the uri doesn't seem right

Reply:
I thought you were wrong at first, as it took me a bit to see the missing path segment (`/pending`). Thanks for pointing that out.